ebuild-writing/common-mistakes/text.xml

```diff
@@ -477,8 +491,42 @@ at least when the ebuild hits the stable branch.
 </p>
 </body>
 </subsection>
-</section>
 
+<subsection>
+<title>Not using latest EAPI</title>
+<body>
+<p>
+Often, user-submitted ebuilds do not use the latest EAPI and may even
+be several versions behind. EAPIs bring new helper functions and improved
+methods for completing common tasks. It's recommended that you port submitted
+ebuilds to the latest EAPI possible (subject to eclass constraints) to improve
+the ebuild's quality and ease review.
+</p>
+</body>
+</subsection>
+
+<subsection>
+<title>Calling pkg-config directly</title>
+<body>
+
+<p>
+You should not call <c>pkg-config</c> directly in ebuilds because this is
+problematic for e.g. cross-compiling. The correct helper respects
+<c>${PKG_CONFIG}</c>. Instead, use <c>tc-getPKG_CONFIG</c> from
+<c>toolchain-funcs.eclass</c>, e.g.
+</p>
+
+<codesample lang="ebuild">
+sed -i -e 's/-lncurses/$($(tc-getPKG_CONFIG) --libs ncurses)/' || die
```

Comment:
This happens to work for ncurses, but generally `pkg-config --libs` may contain slashes in its output:
```
$ pkg-config --libs libpq
-L/usr/lib64/postgresql-13/lib64 -lpq
```
Change the separator to `:` at least (it's still not 100% water-proof, but at least it's safer than `/`).